compiler/options.nim

```diff
@@ -245,7 +255,7 @@ type
     evalTemplateCounter*: int
     evalMacroCounter*: int
     exitcode*: int8
-    cmd*: TCommands  # the command
+    cmd*: CommandRaw  # raw command parsed as enum
```

Comment:
Just call it a Command please, a "raw" command doesn't make much sense :D

Reply:
done